### ISSUE TITLE
Add the ability to specify a "Path Prefix"

### ISSUE THREAD

**Issue by dougmorin** [nylas/N1#454]
Do we, or can we, have the ability to add a Path Prefix in order for email to sync with the server when using IMAP?  Right now my email will not sync with the server in order to show that email has been read, moved, etc.

**Comment by bengotow** [comment]
Hi @dougmorin — there isn't currently a way to specify a path prefix with IMAP. We're planning to expand the IMAP/SMTP options, and I'll see if we can get this in as well.

**Comment by grinich** [comment]
@dougmorin it should still work. What's the IMAP Path Prefix you want to set?

**Comment by dougmorin** [comment]
INBOX

Right now it doesn't sync properly with my current mailbox.  I can't use the app with my work stuff at all.

**Comment by lmartins** [comment]
For me even without a prefix config the general IMAP account works but repeats the prefix on every folder, which is a bit ugly to be honest:

<img width="235" alt="screenshot 2016-03-01 21 14 52" src="https://cloud.githubusercontent.com/assets/151981/13442155/bf570a84-dff2-11e5-800b-5479d88f5444.png">

**Comment by bdimcheff** [comment]
Yeah I have the same problem as @lmartins, but also if I try to create a new folder but don't manually add "INBOX." to the beginning, it fails.  I'm on fastmail.

**Comment by montdidier** [comment]
This would be helpful. At the moment this lack of feature completely snookers me.

**Comment by tokyovigilante** [comment]
I also get mangled folder and subfolder names using Fastmail.
![mangled](https://cloud.githubusercontent.com/assets/12971/14658674/1797c08c-06d9-11e6-8174-d715bb5a4094.jpg)

**Comment by bengotow** [comment]
Hey folks—we'll be fixing the INBOX.prefix issue in the next release (0441acf94aa5c46118759a72fbfe21e33f41d043). There are some edge cases where other custom prefixes are necessary, and we'll be addressing later this year in the sync engine.

**Comment by dougmorin** [comment]
This doesn't actually fix the issue though, this only displays it without the INBOX prefix.  I like what this mail program represents, but I can not use it whatsoever without you adding the ability to actually USE the prefix.  My mail will not sync without it.

**Comment by grinich** [comment]
@dougmorin are you still having this issue?

**Comment by sebastianks** [comment]
@grinich I know this isn't my thread, but this is happening for me to. I can't use the IMAP feature without being able to set IMAP Path Prefix to "INBOX". My folders, except Inbox, won't sync correctly. Will this be fixed?
